avocado-plugins-vt.spec

```diff
@@ -106,7 +106,7 @@ Xunit output, among others.
 %if %{with_python3}
 %package -n python3-%{name}
 Summary: %{summary}
-Requires: python3, python3-devel, python3-avocado >= 51.0, python3-aexpect
+Requires: python3, python3-avocado >= 51.0, python3-aexpect
```

Comment:
any specific reason removing `python3-devel` dependency here?

Reply:
The python3-devel is only used in `passfd.c` so removing it makes sense. Note I have not tried running Avocado-vt without python3-devel but it builds well and fgrep shows only the `passfd.c` requiring it.